lavorare-insieme/dove-quando-e-come.md

```diff
@@ -0,0 +1,101 @@
+# Dove, quando e come
+
+La nostra settimana lavorativa si costituisce di 4 giorni dedicati a lavori per clienti esterni (non
+necessariamente sempre con lo stesso cliente) e di 1 giorno,
+[il venerdì](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md), dedicato
+solo all'apprendimento, ai progetti interni o al contributo open source.
+
+Le giornate lavorative sono di 8 ore piuttosto flessibili. Per legge non si possono comunque
+superare le 13 ore giornaliere e le 48 ore settimanali. L'orario adottato dalla maggior parte
+di noi è compreso tra le 9:30 e le 18:30 considerando un'ora circa per la pausa pranzo.
+L'organizzazione del lavoro deve essere comunque vista nell'ottica del lavoro in team.
+Quindi chi sceglie di lavorare in orari diversi deve essere attento a non impedire ad un
+altro di svolgere correttamente il proprio lavoro.
+
+Riguardo al luogo la flessibilità è ancora maggiore. Nebulab al momento ha due sedi operative, una a
+Pescara e una a Latina, ma si è liberi di lavorare ovunque si voglia (volendo anche da casa
+propria).
+
+Questa flessibilità in termini di orario e luogo è resa possibile dalla disciplina del nostro team e
+dall'uso continuo e attento di strumenti ad hoc.
+
+## Lavorare da remoto
+
+Come menzionato sopra, è possibile in qualunque momento (con l'opportuno preavviso) scegliere di
+lavorare da remoto per un periodo determinato o a tempo indefinito. In Nebulab crediamo fermamente
+nel fatto che tutti dovrebbero lavorare dove sono più produttivi e nel valore aggiunto di una cultura
+incentrata sul lavoro da remoto. Del resto lavoriamo remotamente per quasi tutti i nostri clienti!
```

Comment:
1. remotamente non mi piace :) preferirei una locuzione tipo `in modalità remota` oppure `in questa modalità` visto che `da remoto` è alla fine della frase precedente.
2. per ~quasi~ tutti i nostri clienti.

Reply:
@matteogalliani ho deciso di ripetere "da remoto" alla fine, non mi piacciono tanto le circonlocuzioni alternative.